EIPS/eip-779.md

```diff
@@ -54,6 +54,8 @@ contract WithdrawDAO {
 }
 ```
 
+The 9 blocks including and after the activation block (so block 1,920,000 - 1,920,009) should have "dao-hard-fork" (`0x64616f2d686172642d666f726b`) in the `extraData` field of the block.
```

Comment:
If this was required, then this should be reworded to be a **MUST** statement.

Reply:
If this was not a consensus requirement, then it should not be part of the specification.

Reply:
This is a requirement per the linked article in the EIP: https://blog.slock.it/hard-fork-specification-24b889e70703

Thanks, I reworded it to must =)